full-moon/src/ast/parser_util.rs

```diff
@@ -76,7 +76,7 @@ pub(crate) trait Parser: Sized {
 #[macro_export]
 macro_rules! make_op {
     ($enum:ident, $(#[$outer:meta])* { $($operator:ident,)+ }) => {
-        #[derive(Clone, Debug, Display, PartialEq, Node, Visit)]
+        #[automatically_derived]
```

Comment:
Don't think this change is correct - this derive should still be here?

Reply:
It was a suggestion on resolving the new clippy warnings that didn't pan out. Reverted.